### DISCUSSION TITLE
Initial setup, wrong data entered from developer account.

### DISCUSSION THREAD

**Discussion by eh28574** [astrandb/miele#226] [General]
Hello,

unfortunately I entered the wrong data (from Devpooler) during the initial setup of the integration.
But now, no matter what I try, I can not change this data, I am always redirected to the lagin for My Account. When I enter my data there always comes account link failed. 
How can I change the autofication data (Name, ClienID and ClientSecret).
The integration delete and reinstall I have already tried several times but without success.

**Comment by astrandb** [comment]
The place where you manage application credentials is well hidden in Home Assistant.
Go to Settings->Integrations and klick on the tree-vertical-dot icon in upper right corner and select Application Credentials.

**Reply by eh28574** [reply]
Thank you very much, now everything works.
